src/Queue/Failed/MongoFailedJobProvider.php

```diff
@@ -28,7 +28,7 @@ public function log($connection, $queue, $payload, $exception)
             'connection' => $connection,
             'queue' => $queue,
             'payload' => $payload,
-            'failed_at' => Carbon::now()->getTimestamp(),
+            'failed_at' => new \MongoDB\BSON\UTCDateTime(now()),
```

Comment:
Noted that `Carbon::now()` allows for mocking time in tests, so I presume that `now()` does the same, so we shouldn't rely on creating a new `UTCDateTime` instance without a time given.